app/assets/javascripts/client/controllers/responses/ResponseCtrl.js

```diff
@@ -1,14 +1,38 @@
-PDRClient.controller('ResponseCtrl', [
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('ResponseCtrl', ResponseCtrl);
+
+  ResponseCtrl.$inject = [
     '$scope',
     '$timeout',
-    'SessionService',
-    'Assessment',
     '$stateParams',
-    function($scope, $timeout, SessionService, Assessment, $stateParams) {
-      $scope.user = SessionService.getCurrentUser();
-
-      $scope.assessmentId = $stateParams.assessment_id;
-      $scope.responseId   = $stateParams.response_id;
-      $scope.assessment   = Assessment.get({id: $scope.assessmentId});
-    }
-]);
+    '$modal',
+    'Assessment',
+    'LearningQuestion'
+  ];
+
+  function ResponseCtrl($scope, $timeout, $stateParams, $modal, Assessment, LearningQuestion) {
+
+    $scope.assessmentId = $stateParams.assessment_id;
+    $scope.responseId = $stateParams.response_id;
+    $scope.assessment = Assessment.get({id: $stateParams.assessment_id});
+
+
+    $scope.$on('responses-loaded', function() {
+      $timeout(function() {
+        LearningQuestion.exists({assessment_id: $stateParams.assessment_id})
+            .$promise
+            .then(function() {
+              // No-op - implies that a learning question exists for this user on this assessment
+            }, function() {
```

Comment:
shouldn't we check the error of this call to know if it's exactly 404 what we got here?